packages/strapi-plugin-upload/middlewares/upload/index.js

```diff
@@ -23,6 +24,7 @@ module.exports = strapi => ({
       strapi.app.onerror(err);
     });
 
-    strapi.router.get('/uploads/(.*)', range, koaStatic(staticDir, { defer: true }));
+    const localServerConfig = _.get(strapi, 'plugins.upload.providerOptions.localServer') || {};
```

Comment:
you are missing a config key :) when setting the key into `plugins.js`, the options will be  at `strapi.plugins.upload.config.providerOptions.localServer`

Reply:
Oops. Sorry